torch/_inductor/kernel/mm_plus_mm.py

```diff
@@ -159,6 +160,17 @@ def tuned_mm_plus_mm(mat1, mat2, mat3, mat4, *, layout=None):
     m, n, k, layout, mat1, mat2 = mm_args(mat1, mat2, layout=layout)
     m, n, k, layout, mat3, mat4 = mm_args(mat3, mat4, layout=layout)
 
+    # can not pickle ir.StorageBox for tuning in subprocess. Use its inner
+    # buffer instead.
+    if isinstance(mat1, StorageBox):
```

Comment:
yeah, good catch. I should do this for mm template as well.

I still want to dig a bit more how to make StorageBox pickle'able (I have a unit test show how this fail in test_max_autotune.py). If that works, then we don't need all these. But if that turns out to be a rabbit hole and I can not come up with a good solution, I'll apply these changes to mm template as well.

Reply:
Looks like StorageBox or more specifically the MutableBox can not be pickled since it defines a `__getattr__` method. That confuses pickle. It can be resolved by explicitly defined the `__getstate__`/`__setstate__` method. 

These lines can be deleted now.